src/pages/register/index.js

```diff
@@ -42,9 +42,10 @@ const Register = () => {
             type={"password"}
             />
           </form>
+          <p>{checkPassword(formData.password) ? 'Password OK!' : 'Password must contain at least one capital letter, one number and one special character'}</p>
```

Comment:
This needs to use the translation system

Reply:
This has been fixed